clinica/iotools/converters/genfi_to_bids/genfi_to_bids.py

```diff
@@ -0,0 +1,105 @@
+"""Convert the GENFI dataset into BIDS."""
+
+from os import PathLike
+
+
+def convert_images(
+    path_to_dataset: PathLike, bids_dir: PathLike, path_to_clinical: PathLike, gif: bool
+) -> None:
+    """Convert the entire dataset to BIDS.
+
+    Scans available files in the path_to_dataset,
+    identifies the patients that have images described by the JSON file,
+    converts the image with the highest quality for each category.
+
+    Parameters
+    ----------
+    path_to_dataset: PathLike
+        Path to the raw images
+
+    bids_dir: PathLike
+        Path to directory where the bids will be written
+
+    path_to_clinical: PathLike
+        Path to the clinical data associated with the dataset
+
+    gif: bool
+        If True, indicates the user wants to have the values of the gif parcellation
+    """
+    import os
+
+    import clinica.iotools.bids_utils as bids
+
+    from .genfi_to_bids_utils import (
+        complete_clinical_data,
+        complete_imaging_data,
+        dataset_to_bids,
+        find_clinical_data,
+        intersect_data,
+        read_imaging_data,
+        write_bids,
+    )
+
+    # read the clinical data files
+    if path_to_clinical:
```

Comment:
Can `path_to_clinical` be `None` ? If this is the case, we should have it as an optional parameter.